ocs_ci/utility/vsphere_nodes.py

```diff
@@ -0,0 +1,141 @@
+"""
+Module that contains operations related to vSphere nodes in a cluster
+This module directly interacts with VM nodes
+"""
+import errno
+import logging
+import os
+
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+from ocs_ci.utility.connection import Connection
+from ocs_ci.utility.utils import get_module_ip
+
+logger = logging.getLogger(__name__)
+
+
+class VMNode(object):
```

Comment:
Since it is specific for vSphere, it could be specific too here vSphere():

Reply:
Done